numpy/core/src/multiarray/flagsobject.c

```diff
@@ -702,7 +707,8 @@ static PyObject *
 arrayflags_print(PyArrayFlagsObject *self)
 {
     int fl = self->flags;
-    char * warn_on_write="";
+    char * warn_on_write = "";
```

Comment:
As you're touching this anyway, no space after `*`?

Reply:
Haha, oops, fixed :)